csi.proto

```diff
@@ -871,6 +882,43 @@ message ListVolumesResponse {
   // An empty string is equal to an unspecified field value.
   string next_token = 2;
 }
+message GetVolumeRequest {
+  option (alpha_message) = true;
+
+  // Identity information for a specific volume. This field is
+  // REQUIRED. GetVolume will return with current volume information.
+  string volume_id = 1;
+}
+
+message GetVolumeResponse {
+  option (alpha_message) = true;
+
+  message VolumeStatus{
+    // A list of all `node_id` of nodes that the volume in this entry
+    // is controller published on.
+    // This field is OPTIONAL. If it is not specified and the SP has
+    // the LIST_VOLUMES_PUBLISHED_NODES controller capability, the CO
```

Comment:
Do we want to re-use the same capability (`LIST_VOLUMES_VOLUME_HEALTH`) here, or we want to create a new one `GET_VOLUME_PUBLISHED_NODES`? I'd suggest we create a new one.

Reply:
Added GET_VOLUME_PUBLISHED_NODES.